code/drasil-lang/Language/Drasil/Chunk/Concept/Core.hs

```diff
@@ -34,6 +35,10 @@ instance Idea          ConceptChunk where getA = getA . view idea
 instance Definition    ConceptChunk where defn = dad . defn'
 instance ConceptDomain ConceptChunk where cdom = dad . cdom'
 instance Concept       ConceptChunk where
+instance HasShortName  ConceptChunk where
```

Comment:
same here.

Reply:
please see above

Reply:
Changes were made in d1fb4384458fa7462ddad0ab0163334ca28c5e7e.